test/connect.js

```diff
@@ -24,12 +33,25 @@ test('connect', t => {
     if (id === cherese.id) return 'cherese'
   }
 
-  bob.publish({ type: 'greetings' }, (err) => {
-    t.error(err, 'bob publishes a greeting')
+  t.teardown(() => {
+    alice.close()
+    bob.close()
+    cherese.close()
+  })
+
+  alice.on('rpc:connect', (rpc) => {
+    if (rpc.id === bob.id) t.pass('alice connected to bob!')
+    else if (rpc.id === cherese.id) t.pass('alice connected to cherese!')
+    else t.fail('unknown peer ' + rpc.id)
   })
 
   connect([alice, bob, cherese], { name, friends: true }, (err) => {
-    t.error(err, 'alice an bob connected!')
+    t.error(err, 'happily connected')
+
+    // Test if auto-replication because of friends (follows) + ssb-replicate
+    // is working
+    if (isDB2) return t.end()
+    // currently doesn't work with db2
```

Comment:
will this work after the pr is merged?

Reply:
No. there's lower level problems - basically friends fundamentally changed how it worked. This lower version orchestrated replication with friends on connections... that doesn't work with new db2 for some reasons. Maybe friends as a module is broken as it was flume based at this version. Dunno. Anyway it does what it says "connects" but the rest of the test doesn't